crates/shadowsocks/src/net/option.rs

```diff
@@ -65,6 +65,9 @@ pub struct ConnectOpts {
     /// Outbound socket binds to interface
     pub bind_interface: Option<String>,
 
+    /// Outbound UDP socket allows IP fragmentation
+    pub udp_allow_fragmentation: bool,
```

Comment:
Added the field on `ConnectOpts` [as suggested](https://github.com/shadowsocks/shadowsocks-rust/issues/1780#issuecomment-2497174945) since `UdpSocketOpts` is used for both inbound and outbound sockets, but we could move it to `UdpSocketOpts` instead.

Reply:
Ah, it should be more reasonable to be in `UdpSocketOpts`.